packages/strapi-admin/admin/src/containers/Webhooks/EditView/utils/schema.js

```diff
@@ -0,0 +1,35 @@
+import * as yup from 'yup';
+import { translatedErrors } from 'strapi-helper-plugin';
+import { NAME_REGEX, URL_REGEX } from './fieldsRegex';
+
+const schema = yup.object().shape({
+  name: yup
+    .string(translatedErrors.string)
```

Comment:
The `string` property does not exist in this object. Here's the [corresponding file](https://github.com/strapi/strapi/blob/master/packages/strapi-helper-plugin/lib/src/utils/translatedErrors.js)